packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
 
 ## Configuration
 
+### iconMenu
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
+| menus | Array | Defines which plugins should be rendered as part of the icon menu. |
+| layoutTag | enum NineLayoutTag | Defines where the icon menu is rendered. Only use `NineLayoutTag.TOP_RIGHT` here for the time being. |
+
+Use the configuration during client build.
+
+Example Configuration:
+```js
+{
+  initiallyOpen: 'layerChooser',
+  displayComponent: true,
+  menus: [
+    {
+      plugin: PolarPluginLayerChooser({}),
+      icon: 'fa-layer-group',
+      id: 'layerChooser',
+    },
+    {
+      plugin: PolarPluginDraw({}),
+      icon: 'fa-pencil',
+      id: 'draw',
+      hint: 'Draw or write something on the map'
+    },
+  ],
+  layoutTag: NineLayoutTag.TOP_RIGHT,
+}
+```
+
+### iconMenu.menus
```

Comment:
```suggestion
#### iconMenu.menuEntry
```
See comment further above. If the suggestion above is used, so should this one.

Also, I added a level of nesting, since this is a child element type to `iconMenu`.

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0
0a49a940b56f9097f8e83befe649cc863fe749e1